pkg/log/log.go

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright 2025 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/go-chi/chi/middleware"
```

Comment:
Could we try out Go's new [slog library](https://go.dev/blog/slog) rather than rely on a third-party one?

Reply:
If we're just trying to check code in, we can follow up with this later on also, I'd just omit the logger for now then.

Reply:
done